### PR TITLE
Document submenu api change, `allowPinchZoom` prop move

### PR DESCRIPTION
**Pull request by andy-hook**
https://github.com/radix-ui/primitives/pull/1394
https://github.com/radix-ui/primitives/pull/1400

### CODE REVIEW

File: data/primitives/overview/releases.mdx [hunk 1]
```diff
@@ -23,6 +23,8 @@ metaDescription: Radix Primitives releases and their changelogs.
 
 - Fix issue with native context menu appearing in React 18 <PRLink id={1378} />
 - Add `data-highlighted` attribute to support styling <PRLink id={1388} />
+- [**Breaking**] Improved types when using submenus, they must now be created using explicit parts <PRLink id={1394} />
```
Comment (benoitgrelard) [inline data/primitives/overview/releases.mdx:26]: This makes it seems like the explicit parts were only to improve types slightly, should we mention somehow the nesting issue this resolves? Something like:

"Improve indirect nesting of dropdown menus"

Comment (benoitgrelard) [inline data/primitives/overview/releases.mdx:26]: I think we don't use past tense in the release btw

File: data/primitives/components/context-menu/0.1.7.mdx [hunk 2]
```diff
@@ -307,7 +291,8 @@ The component that pops out in an open context menu.
         </span>
       ),
     },
-  ]}
+
```
Comment (benoitgrelard) [inline data/primitives/components/context-menu/0.1.7.mdx:294]: nit: extra line

File: data/primitives/components/context-menu/0.1.7.mdx [hunk 3]
```diff
@@ -668,11 +615,238 @@ Used to visually separate items in the context menu.
...
+    {
+      name: 'asChild',
+      required: false,
+      type: 'boolean',
+      default: 'false',
+      description:
+        'Change the component to the HTML tag or custom component of the only child. This will merge the original component props with the props of the supplied element/component and change the underlying DOM node.',
+    },
+    {
+      name: 'allowPinchZoom',
```
Comment (benoitgrelard) [inline data/primitives/components/context-menu/0.1.7.mdx:710]: `allowPinchZoom` shouldn't exist at this level right? Or am I missing something?

File: data/primitives/components/dropdown-menu/0.1.7.mdx [hunk 4]
```diff
@@ -733,11 +685,238 @@ Used to visually separate items in the dropdown menu.
...
+    {
+      name: 'asChild',
+      required: false,
+      type: 'boolean',
+      default: 'false',
+      description:
+        'Change the component to the HTML tag or custom component of the only child. This will merge the original component props with the props of the supplied element/component and change the underlying DOM node.',
+    },
+    {
+      name: 'allowPinchZoom',
```
Comment (benoitgrelard) [inline data/primitives/components/dropdown-menu/0.1.7.mdx:780]: `allowPinchZoom` shouldn't exist at this level right? Or am I missing something?

**Review by benoitgrelard — COMMENTED** [review]
I think also we're missing the new CSS variable for the sub.

File: data/primitives/components/context-menu/0.1.7.mdx (hunk 3 above)
Comment (andy-hook) [reply data/primitives/components/context-menu/0.1.7.mdx:710]: Good eyes

File: data/primitives/overview/releases.mdx (hunk 1 above)
Comment (andy-hook) [reply data/primitives/overview/releases.mdx:26]: Solid feedback, I think we should just lead with this and not mention types, that's just an implicit benefit.

Comment (benoitgrelard) [reply data/primitives/overview/releases.mdx:26]: Agreed

Comment (andy-hook) [reply data/primitives/overview/releases.mdx]: Aside: I wonder if we should be placing breaking changes first in the list.

Comment (benoitgrelard) [reply data/primitives/overview/releases.mdx]: Yeah good point, we can do this when we review the final changelog?

**Pull request merged**